### DISCUSSION TITLE
Support FW1906 IC for RGC+CCT strips

### DISCUSSION THREAD

**Discussion by Suxsem** [PlummersSoftwareLLC/NightDriverStrip#614] [Ideas]
**Hi, IC FW1906 is getting very popular due to being able to control RGB+CCT (cold AND warm white) pixels and for accepting 24V power supply.**

Libraries like NeoPixelBus already added support for this IC.

Any chance we can get support in NightDriver too?

Thank you!

**Comment by rbergen** [comment]
I'm converting this to an Idea discussion, as it is clearly not an issue.

**Comment by rbergen** [comment]
I'd say: certainly! Provided you can find (a) developer(s) who are willing to implement such support. :)

**Reply by Suxsem** [reply]
@rbergen does NightDriver has the concept of RGBCCT and it only lacks FW1906 protocol implementation or it totally lacks the RGBCCT concept? thank you

**Comment by robertlipe** [comment]
The email bridge at GitHub seems to be dropping responses today.

See the discussions from, like, #602 and #603 Last week about RGBW[W] LEDs, though that probably applies doubly of these have new timings.

Everything to be said was pretty much said then. We can't do it unless lastled supports it... And it's been at the top of their development TODO list for about three years now with only some very recent progress.

FWIW, i read a LOT about addressible bulbs and don't recall that part number. Just looking at https://gree-leds.com/ic-protocol/fw1906-ic-protocol-and-datasheet.html, it's not directly WS281x compatible, but probably close enough to be in scope. It's not even popular enough to hae been mentioned on the FastLED GitHub, which is what we use for serial LED devices.

In short, I'd expect you'll have some non-trivlal work ahead of you to add them, but we can help you land that work when it' ready.
